blast_config/tx_struct_tng.c

```diff
@@ -661,6 +661,112 @@ channel_t channel_list[] =
     /*---------------------------------------------------------------------------------------*/
 
+    // New star camera channels for TIM MCP
+
+    // Blob-finding parameters
+    // SC1
+    {"sc1_spike_limit",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc1_dynamic_hp",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc1_r_smooth",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc1_high_pass_filter",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc1_r_high_pass_filter",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc1_search_border",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc1_filter_return",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc1_n_sigma",               SCALE(CONVERT_UNITY), TYPE_FLOAT, RATE_1HZ, U_NONE, 0 },
+    {"sc1_unique_star_spacing",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc1_make_static_hp_mask",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc1_use_static_hp_mask",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    // SC2
+    {"sc2_spike_limit",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc2_dynamic_hp",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc2_r_smooth",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc2_high_pass_filter",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc2_r_high_pass_filter",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc2_search_border",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc2_filter_return",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc2_n_sigma",               SCALE(CONVERT_UNITY), TYPE_FLOAT, RATE_1HZ, U_NONE, 0 },
+    {"sc2_unique_star_spacing",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc2_make_static_hp_mask",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+    {"sc2_use_static_hp_mask",               SCALE(CONVERT_UNITY), TYPE_INT16, RATE_1HZ, U_NONE, 0 },
+
+    // Astrometry parameters
+    // SC1
+    // for solving
+    {"sc1_time_limit",               SCALE(CONVERT_UNITY), TYPE_DOUBLE, RATE_1HZ, U_NONE, 0 },
+    {"sc1_logodds",               SCALE(CONVERT_UNITY), TYPE_DOUBLE, RATE_1HZ, U_NONE, 0 },
+    // for ALTAZ
+    {"sc1_latitude",               SCALE(CONVERT_UNITY), TYPE_DOUBLE, RATE_1HZ, U_NONE, 0 },
+    {"sc1_longitude",               SCALE(CONVERT_UNITY), TYPE_DOUBLE, RATE_1HZ, U_NONE, 0 },
+    {"sc1_hm",               SCALE(CONVERT_UNITY), TYPE_DOUBLE, RATE_1HZ, U_NONE, 0 },
```

Comment:
I think this is "height, meters," but it could be clearer. Why not call it altitude? Also, it appears to be relative to the WGS84 ellipsoid, but is it geodetic (above the ellipsoid surface along the surface normal) or geocentric (above the ellipsoid surface along the radius)? I would guess geodetic, but a BLASTcam spelunk seems necessary.

I think a good variable name incorporates both additional pieces of information: altitude and reference datum.

Reply:
we'll see if thats available information somewhere, changed to alt.